fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -3354,7 +3376,7 @@ static class FInstanceExecParam {
         TNetworkAddress host;
         Map<Integer, List<TScanRangeParams>> perNodeScanRanges = Maps.newHashMap();
 
-        int perFragmentInstanceIdx;
+        int perFragmentInstanceIdx; // not useful now
```

Comment:
if not useful，why not another PR delete the code

Reply:
OK